code/game/objects/structures/inflatable.dm

```diff
@@ -147,7 +147,6 @@
 		flick("door_closing",src)
 	sleep(10)
 	density = !density
-	opacity = !opacity
```

Comment:
Why is this being removed here?

Reply:
This is a duplicate that breaks the transparency mechanics of TRANSPARENT walls.

Reply:
Put this information on the PR description then, it will be good to know that this is fixing that too